materials/plot_utils.py

```diff
@@ -0,0 +1,96 @@
+"""Material Property plotting utilities."""
+
+import numpy as np
+from matplotlib import pyplot as plt
+
+
+def plot_property_vs_state(prop, state_name, state_range=None,
+                           value_for_other_state=None, axes=None, **kwargs):
```

Comment:
[taste] Might rename `**kwargs` to `**pyplot_kwargs` or include a comment indicating such

Some people don't like renaming kwargs, I tend to like to name it descriptively

Reply:
fixed in https://github.com/mvernacc/material-properties-interchange/pull/1/commits/df07542479e0732287814c75489dd387e3b32475